tests/cephadm/test_service_stop.py

```diff
@@ -0,0 +1,29 @@
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster):
+    """
+    Verify service stop operation for Mon and Mgr service
+    """
+
+    # Define the service you want to stop
+    service_name = ["mgr", "mon"]
+
+    # Stop the service
+    node = ceph_cluster.get_nodes(role="mon")[0]
+    for i in service_name:
+        log.info(f"Stopping {i} service")
+        out = node.exec_command(sudo=True, cmd=f"cephadm shell ceph orch stop {i}")
+        separator = ", "
+        output = separator.join(out)
+        print(output)
+
+        expected_message = f"Stopping entire {i} service is prohibited."
```

Comment:
If stop operation for this service is prohibited then cli function will raise error. In this situation, exception needs to be catched.